2.wave3_data/2.cellprofiler_ic_processing/README.md

```diff
@@ -0,0 +1,43 @@
+# Create and edit LoadData csv and calculate illumination correction function
```

Comment:
Yes we are calculating IC after the max projection since this is how the process is implied via the conversation with Erin (https://forum.image.sc/t/performing-max-projection-using-phenix-data-based-on-broad-institute-project/77262). It makes sense because why make an IC function using images we aren't going to use.

As well, based on the issue and convo with Erin (https://github.com/CellProfiler/CellProfiler-plugins/issues/179), their example of using this module is using if after IC, so it would make sense we would include this in our analysis pipeline. This will be described in the future README for the 3.cellprofiler_analysis module.